scripts/ai/finish-work.sh

```
@@ -0,0 +1,32 @@
+
+- Start a task with `./scripts/ai/start-work.sh <agent> "<task>"`.
+- Finish a task with `./scripts/ai/finish-work.sh <agent> "<summary>"`.
+- The `commit-msg` hook will append commit activity automatically.
+
+## Activity
+EOF
+fi
+
+{
+  printf "\n### Finish | %s\n" "$timestamp"
+  printf "- Agent: `%s`\n" "$agent"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, printf -  is undefined.

(SC3045)